### PR TITLE
doc: FAQ

### PR DESCRIPTION
**Pull request by sougou**
Signed-off-by: Sugu Sougoumarane <ssougou@gmail.com>

### CODE REVIEW

**Comment by derekperkins** [comment]
The new suggestion auto commit feature is pretty awesome for wording changes.

File: doc/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+In that example, vtgate accepts mysql connections on port 15306, and the authentication info is stored in the json file. So, you should be able to connect to it using the following command:
+
+```
+mysql -h 127.0.0.1 -P 15306 -u mysql_user --password=mysql_password
+```
+
+## Can I override the default db name from `vt_xxx` to my own?
+
+Yes. You can start vttablet with the `-init_db_name_override` command line option to specify a different db name.
```
[the remark this answers is not in the document]
Comment (sougou) [reply doc/FAQ.md]: No downsides that I know of. Added clarification.

**Comment by sougou** [comment]
I did have to fix up the commits later because it doesn't sign them, and it breaks DCO. But it's still good, because I can always amend those in.

**Comment by sougou** [comment]
Going to merge this early, because @lucperkins is migrating to `website`.

**Pull request merged**